README.md

```diff
@@ -74,21 +74,29 @@ Usually the message on the DONE queue will be processed by the captionWF in comm
 docker run --rm --tty --env-file .env sul-speech-to-text --receive
 ```
 
-## The Job File
+## The Job Message Structure
 
-The job file is a JSON object that contains information about how to run Whisper. Minimally it contains the Job ID, which will be used to locate media files in S3 that need to be processed. 
+The job is a JSON object (used as an SQS message payload) that contains information about how to run Whisper. Minimally it contains the Job ID and a list of file names, which will be used to locate media files in S3 that need to be processed.
 
 ```json
 {
-  "id": "8EB51B59-BDFF-4507-B1AA-0DE91ACA388F",
+  "id": "gy983cn1444-v2",
+  "media": [
+    "snl_tomlin_phone_company.mp4"
+  ],
 }
 ```
 
+The convention for the job id is druid-version, since the identifier needs to at least be unique to the druid version to process. But it could also be something more opaque, like a UUID.
+
+The worker will look in the configured S3 bucket for files to process at `"media/{job['id']}/{media_file}"` for each `media_file` in `job["media"]`. E.g. `gy983cn1444-v2/snl_tomlin_phone_company.mp4` for the above example JSON. You can see this illustrated in the `create_job` and `add_media` test functions in `speech_to_text.py`.
```

Comment:
perhaps these added sentences are me being too wordy, happy to trim or drop if you think they're overkill.

Reply:
I think this is good to add, sure. I was kind of hoping to keep the docs independent of SDR wherever possible, if someone else wanted to use it. So maybe flipping the wording?

> The job id must be a unique identifier like a UUID. In some use cases a natural key could be used, as is the case in the SDR where druid-version is used.

Reply:
i like it, will do, thx